lib/ecosystems/enrich_cyclonedx.go

```diff
@@ -160,8 +161,11 @@ func enrichCDXSupplier(comp *cdx.Component, data *packages.Package) {
 					Name: name,
 				}
 				if website, ok := ownerRecord["website"].(string); ok {
-					websites := []string{website}
-					supplier.URL = &websites
+					split := strings.Split(website, ",")
```

Comment:
```suggestion
					split := strings.Split(website, ", ")
```
I'm thinking that splitting on `","` can potentially break a URL with unescaped commas, e.g. `http://host/?foo=bar,baz`. Given ecosyste.ms's behaviour, splitting on `, ` seems safer.